Dbal/MutableAclProvider.php

```diff
@@ -80,6 +80,9 @@ public function createAcl(ObjectIdentityInterface $oid)
      */
     public function deleteAcl(ObjectIdentityInterface $oid)
     {
+        if (!$this->connection->isTransactionActive()) {
+            $this->connection->setNestTransactionsWithSavepoints(true);
+        }
```

Comment:
This would be a very unexpected side-effect, given that the connection is usually shared with the rest of the application.